### ISSUE TITLE
Fails to trace `paraphrase/percent`: Cannot read property 'type' of null

### ISSUE THREAD

**Issue by chrisvariety** [vercel/node-file-trace#125]
@zeit/node-file-trace: 0.6.4
Node: v12.17.0

Input file:

```js
import paraphrase from 'paraphrase/percent';
```

Running:

```js
  const nodeFileTrace = require('@zeit/node-file-trace');
  const files = ['test2.js'];
  const { fileList } = await nodeFileTrace(files);
```

error is:

```
(node:15413) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'type' of null
    at /Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/utils/wrappers.js:324:56
    at Array.every (<anonymous>)
    at handleWrappers (/Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/utils/wrappers.js:324:41)
    at module.exports (/Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/analyze.js:398:3)
    at Job.emitDependency (/Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/node-file-trace.js:245:40)
    at /Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/node-file-trace.js:270:20
    at async Promise.all (index 0)
    at async Job.emitDependency (/Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/node-file-trace.js:251:5)
    at async Promise.all (index 0)
    at async module.exports (/Users/chrismcc/workspace/social-distancin/node_modules/@zeit/node-file-trace/src/node-file-trace.js:27:3)
```

Looks like a potential regression from c7f25ef933989d59af157f1b3e71771e34187d13

**Comment by styfle** [comment]
Thanks for the repro steps!

This is fixed in 0.6.5 and will be fixed in Vercel shortly: https://github.com/vercel/vercel/pull/4667

**Comment by chrisvariety** [comment]
@styfle awesome, thanks so much! ✨
